tests/rptest/tests/flink_basic_test.py

```diff
@@ -214,14 +229,39 @@ def test_transaction_workload(self):
         #     /workloads/data/part-cfbe2720-7117-40fd-8c28-fca31a459ff8-0-0
         # Data sample:
         # ...
-        # 201,"2024-01-05 20:38:53.125",46
+        # 542,415,31,"2024-01-09 22:22:43.647",17,TiJJtOuzOkiOsGmws
         # ...
-        files = self.flink.node.account.ssh_output(f"ls -1 {_data_path}")
-        files = files.decode().splitlines()
+
+        # Wait for file, timeout 5 min (10 sec * 30 iterations)
+        self.logger.info("Waiting for consume workload data file")
+        backoff_sec = 10
+        iterations = 30
+        files = []
+        while iterations > 0:
```

Comment:
nit: We have a utility for this pattern.. look for wait_until in ducktape tests

Reply:
Will do.